scripts/uberenv/packages/serac/package.py

```diff
@@ -97,24 +96,31 @@ class Serac(CMakePackage):
     depends_on('py-sphinx', when="+devtools")
 
     # Libraries that support +debug
-    depends_on("mfem~shared~zlib+hypre+metis+superlu-dist+lapack+mpi")
-    depends_on("mfem~shared~zlib+hypre+metis+superlu-dist+lapack+mpi+debug", when="+debug")
-    depends_on("hypre~shared~superlu-dist+mpi")
-    depends_on("hypre~shared~superlu-dist+mpi+debug", when="+debug")
+    debug_deps = ["mfem@4.0.0~shared~zlib+hypre+metis+superlu-dist+lapack+mpi",
```

Comment:
should this be mfem@4.1.0?

Reply:
Yep. Thanks for the catch!

Reply:
Yeah, turns out I didn't push it. Whoops!